beamer/agent/state_machine.py

```diff
@@ -85,35 +85,34 @@ def process_event(event: Event, context: Context) -> HandlerResult:
     if isinstance(event, LatestBlockUpdatedEvent):
         return _handle_latest_block_updated(event, context)
 
-    elif isinstance(event, RequestCreated):
+    if isinstance(event, RequestCreated):
         return _handle_request_created(event, context)
 
-    elif isinstance(event, RequestFilled):
+    if isinstance(event, RequestFilled):
         return _handle_request_filled(event, context)
 
-    elif isinstance(event, DepositWithdrawn):
+    if isinstance(event, DepositWithdrawn):
         return _handle_deposit_withdrawn(event, context)
 
-    elif isinstance(event, ClaimMade):
+    if isinstance(event, ClaimMade):
         return _handle_claim_made(event, context)
 
-    elif isinstance(event, ClaimStakeWithdrawn):
+    if isinstance(event, ClaimStakeWithdrawn):
         return _handle_claim_stake_withdrawn(event, context)
 
-    elif isinstance(event, RequestResolved):
+    if isinstance(event, RequestResolved):
         return _handle_request_resolved(event, context)
 
-    elif isinstance(event, FillInvalidated):
+    if isinstance(event, FillInvalidated):
         return _handle_fill_invalidated(event, context)
 
-    elif isinstance(event, FillInvalidatedResolved):
+    if isinstance(event, FillInvalidatedResolved):
         return _handle_fill_invalidated_resolved(event, context)
 
-    elif isinstance(event, ChainUpdated):
+    if isinstance(event, ChainUpdated):
         return _handle_chain_updated(event, context)
 
-    else:
-        raise RuntimeError("Unrecognized event type")
+    raise RuntimeError("Unrecognized event type")
```

Comment:
Ugh, I see where this is coming from but it's... meh.
I'm thinking we should rather convert the entire thing to a `match` statement.